packages/components/app/styles/components/tabs.scss

```diff
@@ -9,14 +9,14 @@ $indicator-height: 3px;
 .hds-tabs {}
 
 // Sub-components:
-// TODO: MOVE!
+// TODO: MOVE?
 .hds-tab-list {
   padding-bottom: round(calc($indicator-height / 2));
   // overflow-x: auto;
   // -webkit-overflow-scrolling: touch;
   // white-space: nowrap; // move
 
-  // Un-nest after moving
+  // Un-nest?
```

Comment:
I can start refactoring class names now as people recommend...

My preference for BEM child class naming is to use sass nested structure like:
```
.parent-name {
  color: red;

  &--child-name {font-weight: bold;}
}
```
but if we don't like to use that I could unnest this to simplify the rendered css selectors.